relationship_extractor.md

```diff
@@ -0,0 +1,105 @@
+# What works for relationship extraction API
+
+## Relationship 
+What works
+
+```
+"Family of P13"
+"Son of P12 and Daughter of P12"
+"Friend of P12, P12, 14 and son of P13"
+```
+
+What doesn't work : Relationship without word `of`
+```
+"Friend: P12"
+"Friends with P12"
+"Wife of p12"
+```
+
+## Travel Places
+What works
+```
+Travelled from Wuhan
+Travel history Italy, Pune, Mumbai
+Travlled from South Korea via Mumbai
+```
+
+## Nationality 
+What works 
+```
+Indian student studying in wuhan
+Italian tourise
+```
+What doesn't work
+```
+Indian travelling with Italian tourist
+# Will return Indian and Italian
+```
+
+## API Details 
+API URL : http://coronatravelhistory.pythonanywhere.com/
+
+Example request :
+```
+curl --header "Content-Type: application/json" --request POST --data '{"patients":[{"patientId":"1","notes":"Student from Wuhan, recovery confirmed on 14 Feb"},{"patientId":"2","notes":"Family members of P4"},{"patientId":"3","notes":"Travel from Italy on 29/02/2020 through Doha"}]}' http://coronatravelhistory.pythonanywhere.com/
```

Comment:
Add a Python example as well? Use Postman and share that please?

Reply:
96a45f5ed0c6d759e88ac9821c86bf407fd2b884 solves this